prediction_market_agent/tools/google_search.py

```diff
@@ -4,7 +4,7 @@
 from prediction_market_agent import utils
 
 
-def google_search(query: str) -> t.List[str]:
```

Comment:
does mypy not like `t.List` or `t.Dict`?

Reply:
Sorry! It's my old habit to switch `t.X` to just `x[]`.

Do you prefer `t.List[..]`? Doing `list[...], set[...], dict[...]` is available from Python 3.9 without the need to import `typing`.

Reply:
I guess I've been doing e.g. `t.List` instead of `list`, but I don't have a strong preference - we can just pick one. I'm happy to go with python types!